core/src/avm2/globals/flash/ui/ContextMenu.as

```diff
@@ -32,9 +35,25 @@ package flash.ui
             this._builtInItems = value;
         }
 
-        public static function get isSupported() : Boolean
-        {
+        public function get clipboardItems(): ContextMenuClipboardItems {
+            return this._clipboardItems;
+        }
+
+        public function set clipboardItems(value:ContextMenuClipboardItems):void {
+            this._clipboardItems = value;
```

Comment:
This should have a stub warning?

Reply:
https://github.com/ruffle-rs/ruffle/pull/14038#discussion_r1395934331

Dinnerbone said that if things set what they should set, they shouldn't be stubs if if they do nothing. Difference in philosophy I guess. Probably a discussion for Discord.

Reply:
Disagree, this code is doing what it's supposed to. The actual functionality is not here.